drivers/serial/Kconfig.nrfx_uart_instance

```diff
@@ -0,0 +1,95 @@
+#nRF UART(E) instance configuration
+
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if $(nrfx_uart_num) = 0
+config UART_$(nrfx_uart_num)_NRF_UART
+	def_bool HAS_HW_NRF_UART$(nrfx_uart_num)
+	depends on DT_HAS_NORDIC_NRF_UART_ENABLED
+	help
+	  Enable nRF UART without EasyDMA on port $(nrfx_uart_num).
+endif
+
+config UART_$(nrfx_uart_num)_NRF_UARTE
+	def_bool HAS_HW_NRF_UARTE$(nrfx_uart_num)
+	depends on DT_HAS_NORDIC_NRF_UARTE_ENABLED
+	help
+	  Enable nRF UART with EasyDMA on port $(nrfx_uart_num).
+
+if UART_$(nrfx_uart_num)_NRF_UART || UART_$(nrfx_uart_num)_NRF_UARTE
```

Comment:
These options were removed in aa7d675935517cce0668fa12491602c321b1a0bf and should not be restored. Use `HAS_HW_NRF_UARTx` and `HAS_HW_NRF_UARTEx` directly instead.

Reply:
done, however, i still had to use as intermediate kconfig `UART_x_NRF_UARTS` because cannot use HAS_HW_NRF_UARTx in the template because it exists only for instance 0.